COLLADA2GLTF/COLLADA2GLTFWriter.cpp

```diff
@@ -95,7 +95,7 @@ namespace GLTF
         else
         {
             // We have the data in memory so use it
-            if (!root.loadDocument(asset->getInputFilePath(), fileData.data(), fileData.size())) {
+            if (!root.loadDocument(asset->getInputFilePath(), fileData.data(), (int)fileData.size())) {
```

Comment:
Were you seeing a warning here? I don't see one locally and I can't think of a reason why this would need to be cast.